tests/unit/Model/RamlModelTest.php

```diff
@@ -479,6 +479,9 @@ public function getModelClasses($searchPath)
             if (strpos($class, 'Core\\Helper') > 0) {
                 continue;
             }
+            if (strpos($class, 'Core\\Model\\Message') > 0) {
```

Comment:
This disables some of the tests regarding messages. The correct way would be to add the failing message definition to the api specs

Reply:
I didn't realize that the Message didn't exist in the API specs ;) . I just open the PR with the changes